nova/scheduler/filters/shard_filter.py

```diff
@@ -14,6 +14,7 @@
 #    under the License.
 from oslo_log import log as logging
 
+from nova.bigvm.manager import BIGVM_EXCLUSIVE_TRAIT
```

Comment:
I think we should not import anything from `nova.bigvm` into general code. Let's move this to `nova.utils` if we need it outside of `nova-bigvm`.